topdown/trace.go

```diff
@@ -196,6 +206,24 @@ func formatEventSpaces(event *Event, depth int) int {
 	return depth + 1
 }
 
+func formatLocation(event *Event) string {
+	if event.Op == NoteOp {
+		return fmt.Sprintf("%-19v", "note")
+	}
+
+	location := event.Location
+	if location == nil {
+		// Most likely a trace event
+		return fmt.Sprintf("%-19v", "trace")
```

Comment:
nit: Would it make sense to just omit `trace` and just print spaces? I want to avoid any confusion if someone sees the pretty output and there is a `trace` location it might be unclear what that means versus empty which implies it doesn't have a location.

Reply:
Indeed, it makes sense to leave it empty if there is no location known.